frontend/src/Components/Form/FormInputButton.tsx

```diff
@@ -0,0 +1,45 @@
+import classNames from 'classnames';
+import React from 'react';
+import Button from 'Components/Link/Button';
+import { LinkProps } from 'Components/Link/Link';
+import SpinnerButton from 'Components/Link/SpinnerButton';
+import { kinds } from 'Helpers/Props';
+import styles from './FormInputButton.css';
+
+export interface FormInputButtonProps extends Omit<LinkProps, 'children'> {
+  children: Required<LinkProps['children']>;
+  canSpin?: boolean;
+  isLastButton?: boolean;
+}
```

Comment:
`SpinnerButton` builds on top of `Button` so this can be simplified to:

```suggestion
export interface FormInputButtonProps extends ButtonProps {
  canSpin?: false;
  isLastButton?: boolean;
}
```